CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## MASTER
 
+## Dotenv 0.9.0.0
+* Remove `loadSafeFile`
```

Comment:
@CristhianMotoche maybe we could expand a little bit more here about the reasoning behind this change?